backend/btrixcloud/operator.py

```diff
@@ -1137,6 +1200,71 @@ async def do_crawl_finished_tasks(
         # finally, delete job
         await self.delete_crawl_job(crawl_id)
 
+    async def compute_execution_time(self, crawl_id, oid, crawl=None):
```

Comment:
Another option would be to have this run incrementally, as each `:end` value is added, look for corresponding start and add that up? The files added list sort of works like that.
Though, its about the same, but we would also want to handle if scale is adjusted down.